Qa calvin

Calvinjwala opened the pull request:
- Fixed layout for the impact, calculator, and overview page
- Added net-new assets and replaced some older ones
- Style changes per requests in the QA doc
- Copy changes per Figma

mDuo13 committed:
Consistent use of AccountID in transaction types

mDuo13 committed:
Merge pull request #899 from mDuo13/accountid_consistency

Consistent use of AccountID in transaction types

Calvinjwala committed:
updates to all the QA requests. Fixed layout for the impact, calculator, and overview pages. Added net-new assets that have been updated or were missing. Removed unnecessary styling and added more helpers.

Calvinjwala committed:
minor class change on calc page

Calvinjwala committed:
Merge branch 'master' of github.com:ripple/xrpl-dev-portal into qa-calvin

Calvinjwala committed:
adding the social links to the social share buttons. minor edit for the toggle buttons

Calvinjwala committed:
edits to the toggle buttons

Calvinjwala committed:
toggle button edits for mobile

mDuo13 commented on tool/template-learn-uses.html:
```diff
@@ -72,22 +72,22 @@ <h2 class="mb-10">{% trans %}Digital Cryptocurrency Wallets{% endtrans %}</h2>
     </section>
 
-    <section class="row last-section">
+    <section class="row mb-50">
```
This undoes some changes I put in specifically to make the spacing at the bottom of the page match Gratas' request. Is that intentional?

Calvinjwala replied:
Did not know that. We can leave as is, but the spacing is more or less the same with one less class

mDuo13 commented on tool/template-calculator.html:
```diff
@@ -16,35 +16,38 @@
-      <div class="col-sm-4 d-sm-none d-md-block sustainability-bg">
-        &nbsp;
+      <div class="col-sm-4 d-sm-none d-md-block">
+        <img class="mw-100" src="./img/green/green-graphic.png">
```
For decorative images such as this one, we should always [provide empty alt text so that screen readers do not read the file name aloud](https://www.w3.org/WAI/tutorials/images/decorative/).

(There are many other cases of this throughout the new pages; we should comb through and fix them at some point.)

Calvinjwala replied:
Sounds good, I can take a look at that

mDuo13 reviewed (changes requested):
Left a couple comments. I plan to take a closer look tomorrow.

mDuo13 committed:
Change "broken" calculator anchors to data attrs

Anchor links to d-crypto etc. were not really broken but were used in
JavaScript; this changes them to data attributes so the link checker
doesn't report them as broken.

mDuo13 committed:
Merge pull request #902 from mDuo13/calc-links

Change "broken" calculator anchors to data attrs

Calvinjwala committed:
fixed github reviewscomments

mDuo13 commented on tool/template-learn-uses.html:
```diff
@@ -82,7 +82,7 @@ <h2 class="mb-10">{% trans %}Exchanges{% endtrans %}</h2>
     </section>
 
-    <section class="row mb-50">
+    <section class="row lsat-section">
```
Typo?

mDuo13 commented on tool/template-calculator.html:
```diff
@@ -25,7 +25,7 @@
       <h6 class="section-marker">{% trans %}Sustainability{% endtrans %}</h6>
       <div class="col-sm-4 d-sm-none d-md-block">
-        <img class="mw-100" src="./img/green/green-graphic.png">
+        <img class="mw-100" src="./img/green/green-graphic.png" alt="{% trans %}Sustainability icon{% endtrans %}">
```
I think for cases like this it's better to have an actually empty alt text since the image is purely decorative. (Read the link from my previous comment for details)

Calvinjwala committed:
fixing typo
